src/Upgrade/Application/Provider/ConfigurationProviderInterface.php

```diff
@@ -49,6 +49,14 @@ public function getUpgradeStrategy(): string;
      */
     public function getComposerInstallDependencies(): bool;
 
+    /**
+     * Specification:
+     * - Defines trigger Integrator command integrator:manifest:run
```

Comment:
code fencing using ` should be used around code snippets etc.

Reply:
is the correct result `Defines trigger Integrator command `integrator:manifest:run``?